tests/worker/test_workflow.py

```diff
@@ -7973,3 +7973,32 @@ async def test_quick_activity_swallows_cancellation(client: Client):
             assert cause.message == "Workflow cancelled"
 
         temporalio.worker._workflow_instance._raise_on_cancelling_completed_activity_override = False
+
+
+@activity.defn
+def use_in_workflow():
```

Comment:
```suggestion
def use_in_workflow() -> bool:
```

And return from workflow and assert the workflow result is `False` in the test, but not technically a blocking requirement